### PR TITLE
Add URI decoding to file names.

### PR DESCRIPTION
**Pull request by muri9**

### CODE REVIEW

File: src/coffee/dropship_file.coffee [hunk 1]
```diff
@@ -105,7 +105,7 @@ class DropshipFile
     basename = @url.split('#', 2)[0].split('?', 2)[0]
     while basename.substring(basename.length - 1) == '/'
       basename = basename.substring 0, basename.length - 1
-    basename.substring basename.lastIndexOf('/') + 1
+    decodeURIComponent(basename.substring basename.lastIndexOf('/') + 1)
```
Comment (pwnall) [inline src/coffee/dropship_file.coffee]: Can you please put the parenthesis after `basename.substring`, so it applies to the inner call?

**Comment by pwnall** [comment]
Thank you, @Shiawase! I'll merge as soon as you move the parenthesis.

Alternatively, if you don't want to deal with it, I can do that for you, just let me know!

**Comment by muri9** [comment]
Is that right? I am a new to coffeescript. I tried to build it, works ok.

**Comment by pwnall** [comment]
Thank you, @Shiawase!

The parser will accept it both ways. It's a matter of keeping the code style consistent.

I merged your change and pushed an update to the Chrome store.

**Comment by muri9** [comment]
Awesome, thank you )

**Pull request merged**